lib/lita/handlers/jenkins.rb

```diff
@@ -77,11 +77,9 @@ def jenkins_list(response)
       end
 
       def headers
-        headers = {}
-        if auth = config.auth
-          headers["Authorization"] = "Basic #{Base64.encode64(auth).chomp}"
+        {}.tap do |headers|
+          headers["Authorization"] = "Basic #{Base64.encode64(config.auth).chomp}" if config.auth
```

Comment:
This is just a personal preference so no need to change if you don't want to...but I do not like trailing `if` conditions if the code before it is long. Check the readability between the two:

`this_is_some_very_long_method_so_the_condition_gets_pushed_far_to_the_right if true`

```
if true
  this_is_some_very_long_method_so_the_condition_gets_pushed_far_to_the_right
end
```

In the first case it's easy to miss the condition if you're scanning the code quickly. The second version makes it much easier to see the conditional.